src/drivers/common/memory.h

```diff
@@ -11,15 +11,24 @@
 
 #include <stdint.h>
 
+#define MAX_SEGMENTS 64
+
 struct periph_memory_desc {
 	uint32_t start;
 	uint32_t len;
 };
 
+struct _segment {
```

Comment:
This struct has almost the same content as `struct periph_memory_desc`, are you sure it's necessary to add one more structure?

Reply:
Ok, I removed the structure _segment from this file.